### PR TITLE
PPF-314 - Show price total and coupon information

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added
- [Show price overview in activation dialog](https://github.com/cultuurnet/publiq-platform/commit/f799d0fc11b63985a373cc56481f790cf740fbf9)
- [Show coupon info on detail page](https://github.com/cultuurnet/publiq-platform/commit/b345037cdf69fec46752613886e4e5fff173b18d)

---
Ticket: https://jira.uitdatabank.be/browse/PPF-314

Screenshots:

Screenshot 2024-03-27 at 17 31 28
Screenshot 2024-03-27 at 17 00 34

### CODE REVIEW

**Review by LucWollants — APPROVED** [review]
Visually it could be a good idea to merge the top plan information with the bottom detail price information.

File: resources/ts/Components/ActivationDialog.tsx [hunk 1]
```diff
@@ -11,6 +11,59 @@ import { useIsMobile } from "../hooks/useIsMobile";
...
+            <span className="text-publiq-orange">
+              {`- ${formatCurrency(
+                subscription.currency,
+                couponInfo.reductionAmount
+              )}`}
+            </span>
+          </>
+        )}
+
+        <span>{`${pricingPlan.title} ${t("integrations.activation_dialog.price_overview.plan")}`}</span>
```
Comment (brampauwelyn) [inline resources/ts/Components/ActivationDialog.tsx:54]: Is there a reason why you need template literals here?
```suggestion
        <span>{pricingPlan.title} {t("integrations.activation_dialog.price_overview.plan")}</span>
```

File: resources/ts/Components/ActivationDialog.tsx [hunk 2]
```diff
@@ -11,6 +11,59 @@ import { useIsMobile } from "../hooks/useIsMobile";
...
+  const couponInfo = useContext(CouponInfoContext);
+
+  return (
+    <section className="flex flex-col flex-1 gap-2 border-t mt-4 pt-4 text-sm">
+      <Heading level={4}>
+        {t("integrations.activation_dialog.price_overview.title")}
+      </Heading>
+
+      <div className="grid grid-cols-2 gap-1">
+        <span>{`${pricingPlan.title} ${t("integrations.activation_dialog.price_overview.setup")}`}</span>
```
Comment (brampauwelyn) [inline resources/ts/Components/ActivationDialog.tsx:37]: Is it necessary to use template literals here? 
```suggestion
        <span>{pricingPlan.title} {t("integrations.activation_dialog.price_overview.setup")}</span>
```

**Review by brampauwelyn — APPROVED** [review]

**Comment by simon-debruijn** [comment]
Updated price overview design

Screenshot 2024-03-28 at 15 58 00

File: resources/ts/Components/ActivationDialog.tsx (hunk 2 above)
Comment (simon-debruijn) [reply resources/ts/Components/ActivationDialog.tsx]: This is to avoid jsx `{" "}`

**Pull request merged**